Build failing on alpine edge with v0.6.1

Relates to ffbuilds/static-librav1e#4

**Describe the bug**
Please see the console output below for the error. The builds were working with alpine edge prior versions 0.5.1 and below. See https://github.com/ffbuilds/static-librav1e/blob/main/Dockerfile for details.

**To Reproduce**
[Steps to reproduce the behavior](https://github.com/ffbuilds/static-librav1e/actions/runs/3591186746/jobs/6045420946)

**Expected behavior**
I expect the build to pass

**Required Information**
Toolchain (if is a build problem):
```
$ cargo --version cargo (1.65.0-r0)
$ rustc --version rustc_version v0.4.0
$ nasm --version # if on x86_64 nasm (2.15.05-r1)
```
Version:
```
$ rav1e --version 0.6.1
```
Operating system:
```
$ uname -a ALPINE_VERSION=edge
```

**Console Output**
Please paste the console output below if it is relevant,
```
#14 206.2    Compiling rav1e v0.6.1 (/tmp/rav1e)
#14 299.1 error: prefix `Linux` is unknown
#14 299.1   --> /tmp/rav1e/target/x86_64-alpine-linux-musl/release/build/rav1e-baf8555d2aa5ac72/out/built.rs:72:57
#14 299.1    |
#14 299.1 72 | pub const RUSTC_VERSION: &str = r"rustc 1.65.0 ("Alpine Linux")";
#14 299.1    |                                                         ^^^^^ unknown prefix
#14 299.1    |
#14 299.1    = note: prefixed identifiers and literals are reserved since Rust 2021
#14 299.1 help: consider inserting whitespace here
#14 299.1    |
#14 299.1 72 | pub const RUSTC_VERSION: &str = r"rustc 1.65.0 ("Alpine Linux ")";
#14 299.1    |                                                              +
#14 299.1 
#14 299.1 error: prefix `Linux` is unknown
#14 299.1   --> /tmp/rav1e/target/x86_64-alpine-linux-musl/release/build/rav1e-baf8555d2aa5ac72/out/built.rs:75:61
#14 299.1    |
#14 299.1 75 | pub const RUSTDOC_VERSION: &str = r"rustdoc 1.65.0 ("Alpine Linux")";
#14 299.1    |                                                             ^^^^^ unknown prefix
#14 299.1    |
#14 299.1    = note: prefixed identifiers and literals are reserved since Rust 2021
#14 299.1 help: consider inserting whitespace here
#14 299.1    |
#14 299.1 75 | pub const RUSTDOC_VERSION: &str = r"rustdoc 1.65.0 ("Alpine Linux ")";
#14 299.1    |                                                                  +
#14 299.1 
#14 299.1 error: suffixes on a string literal are invalid
#14 299.1   --> /tmp/rav1e/target/x86_64-alpine-linux-musl/release/build/rav1e-baf8555d2aa5ac72/out/built.rs:72:33
#14 299.1    |
#14 299.1 72 | pub const RUSTC_VERSION: &str = r"rustc 1.65.0 ("Alpine Linux")";
#14 299.1    |                                 ^^^^^^^^^^^^^^^^^^^^^^^ invalid suffix `Alpine`
#14 299.1 
#14 299.1 error: expected one of `.`, `;`, `?`, or an operator, found `Linux`
#14 299.1   --> /tmp/rav1e/target/x86_64-alpine-linux-musl/release/build/rav1e-baf8555d2aa5ac72/out/built.rs:72:57
#14 299.1    |
#14 299.1 72 | pub const RUSTC_VERSION: &str = r"rustc 1.65.0 ("Alpine Linux")";
#14 299.1    |                                                         ^^^^^ expected one of `.`, `;`, `?`, or an operator
#14 299.1 
#14 299.7 error[E0425]: cannot find value `GIT_COMMIT_HASH` in module `crate::built_info`
#14 299.7    --> src/lib.rs:417:24
#14 299.7     |
#14 299.7 417 |     crate::built_info::GIT_COMMIT_HASH.unwrap_or_default().to_string()
#14 299.7     |                        ^^^^^^^^^^^^^^^ not found in `crate::built_info`
#14 299.7 
#14 299.7 error[E0425]: cannot find value `GIT_VERSION` in module `crate::built_info`
#14 299.7    --> src/lib.rs:429:26
#14 299.7     |
#14 299.7 429 |       crate::built_info::GIT_VERSION.unwrap_or_default()
#14 299.7     |                          ^^^^^^^^^^^ help: a constant with a similar name exists: `PKG_VERSION`
#14 299.7     |
#14 299.7    ::: /tmp/rav1e/target/x86_64-alpine-linux-musl/release/build/rav1e-baf8555d2aa5ac72/out/built.rs:9:1
#14 299.7     |
#14 299.7 9   | pub const PKG_VERSION: &str = r"0.6.1";
#14 299.7     | --------------------------------------- similarly named constant `PKG_VERSION` defined here
#14 299.7 
#14 303.9 For more information about this error, try `rustc --explain E0425`.
#14 303.9 error: could not compile `rav1e` due to 6 previous errors
#14 303.9 Error: CliError { error: Some(1 job failed), exit_code: 101 }
#14 ERROR: process "/bin/sh -c apk add --no-cache --virtual build     rust cargo git pkgconf openssl-dev nasm cargo-c &&   RUSTFLAGS=\"-C target-feature=+crt-static\" cargo cinstall --release &&   pkg-config --exists --modversion --path rav1e &&   ar -t /usr/local/lib/librav1e.a &&   readelf -h /usr/local/lib/librav1e.a &&   apk del build" did not complete successfully: exit code: 1
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -119,7 +119,7 @@ features = ["png"]
 [build-dependencies]
 cc = { version = "1.0", optional = true, features = ["parallel"] }
 rustc_version = "0.4"
-built = { version = "0.5.1", features = ["chrono"] }
+built = { version = "0.5.2", features = ["chrono"] }
 
 [build-dependencies.nasm-rs]
 version = "0.2"

